HW1.py

```diff
@@ -0,0 +1,120 @@
+import sys
+
+import pandas as pd
+import plotly.express as px
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.tree import DecisionTreeClassifier
+
+
+def main():
+
+    # Load Data
+    iris_data = pd.read_csv("iris.data")
+
+    # Rename columns
+    iris_data = iris_data.rename(
+        columns={
+            "5.1": "sepal_length",
+            "3.5": "sepal_width",
+            "1.4": "petal_length",
+            "0.2": "petal_width",
+        }
```

Comment:
You may want to add an extra entry. in this dict.  Otherwise, the last column wouldn't have a descriptive name.

Reply:
Fixed.

Reference: ae50f75027bf6f9296028511081cfe43ebea0912